File: examples/go/db/read/readRecords.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Read data from a table. Lookup can be by ID or via querying any field in the record.
func ReadRecords() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.Read(&db.ReadRequest{
		Query: "age == 43",
		Table: "users",
	})
	fmt.Println(rsp, err)
}

File: examples/js/db/read/readRecords.js
const { DbService } = require("m3o/db");

// Read data from a table. Lookup can be by ID or via querying any field in the record.
async function readRecords() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.read({
    query: "age == 43",
    table: "users",
  });
  console.log(rsp);
}

readRecords();
